components/sup/src/util/path.rs

```diff
@@ -123,6 +124,24 @@ pub fn busybox_paths() -> Result<Vec<PathBuf>> {
     Ok(bb_paths)
 }
 
+#[cfg(target_os = "windows")]
+pub fn interpreter_paths() -> Result<Vec<PathBuf>> {
+    let empty: Vec<PathBuf> = Vec::new();
+    Ok(empty)
+}
+
+pub fn append_interpreter_and_path(orig_paths: &mut Vec<PathBuf>) -> Result<String> {
+    let mut paths = try!(interpreter_paths()).to_owned();
+    orig_paths.append(&mut paths);
+    if let Some(val) = env::var_os("PATH") {
+        let mut os_paths = env::split_paths(&val).collect::<Vec<PathBuf>>().to_owned();
```

Comment:
heh. you are right. Probably another case where it fixed something before the rest of my keyboard mashing that fixed it.